ecl/eclagent/eclagent.cpp

```diff
@@ -2020,12 +2012,6 @@ void EclAgent::doProcess()
                 UWARNLOG(errCode, "%s", rmMsg.str());
         }
 
-        if (globals->getPropBool("DUMPFINALWU", false))
```

Comment:
Why is this option being removed?

Reply:
It was undocumented and not especially useful (IMHO)